CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivities.uc

```diff
@@ -748,11 +770,12 @@ static function string GetUnitDetails (XComGameState_Activity ActivityState, XCo
 		}
 		else
 		{
-			UnitString = class'X2StrategyElement_DefaultRewards'.default.DoctorPrefixText @ UnitState.GetName(eNameType_Full) @ "-" @ RewardState.GetMyTemplate().DisplayName;
+			UnitString = class'X2StrategyElement_DefaultRewards'.default.DoctorPrefixText @ UnitState.GetName(eNameType_Full) @ "-" @ MissionRewardState.GetMyTemplate().DisplayName;
```

Comment:
I'm slightly confused at the `@ "-" @ MissionRewardState.GetMyTemplate().DisplayName` part. Even in your screenshot, it's empty:

![image](https://user-images.githubusercontent.com/2865341/75322882-5c8aa300-587c-11ea-91ac-c91f9c74e874.png)

Reply:
When I ran the test that got that screenshot, I had forgotten to change `RewardState` to `MissionRewardState` on that line, hence a missing "Engineer" string there.

Reply:
ok, I'll trust your word 😄